packages/widgets/package.json

```diff
@@ -20,17 +20,16 @@
     "peerDependencies": {
         "react": "^16.8",
         "react-dom": "^16.8",
-        "styled-jsx": "^3.2"
+        "styled-jsx": "^3.2",
+        "@dhis2/ui-core": "^5.0",
```

Comment:
This package was at `5.3` previously. Do we really wanna have it as `5.0` here?

Reply:
Again it shouldn't really matter, since they're compatible, but I'll use `5.3` for consistency nice catch